src/BusyIndicator/BusyIndicator.js

```diff
@@ -0,0 +1,77 @@
+import { BUSY_INDICATOR_SIZES } from '../utils/constants';
+import classnames from 'classnames';
+import CustomPropTypes from '../utils/CustomPropTypes/CustomPropTypes';
+import PropTypes from 'prop-types';
+import React, { useEffect } from 'react';
+
+const BusyIndicator = React.forwardRef(({
+    children,
+    className,
+    show,
+    size,
+    disableStyles,
+    localizedText,
+    ...props
+}, ref) => {
+
+    useEffect(() => {
+        if (!disableStyles) {
+            require('fundamental-styles/dist/icon.css');
+            require('fundamental-styles/dist/busy-indicator.css');
+        }
+    }, []);
+
+    const busyIndicatorClasses = classnames(
+        {
+            [`fd-busy-indicator--${size}`]: size === 'm' || size === 'l',
+            'fd-busy-indicator': size === 's'
+        },
+        className
+    );
+
+    if (!show)
+        return null;
+
+    return (
+        <div
+            {...props}
+            aria-hidden='false'
+            aria-label={localizedText.loading}
+            className={busyIndicatorClasses}
+            ref={ref} >
+            <div className='fd-busy-indicator--circle-0' />
+            <div className='fd-busy-indicator--circle-1' />
+            <div className='fd-busy-indicator--circle-2' />
+        </div>
+    );
+});
+
+BusyIndicator.displayName = 'BusyIndicator';
+
+BusyIndicator.propTypes = {
+    children: PropTypes.node,
```

Comment:
It looks like you can take children out of the propTypes, it isn't used

Reply:
Done.